src/screens/EventsScreen/ResetAllFiltersButton.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+import React from "react";
+import { Animated, StyleSheet, Easing } from "react-native";
+import type { ViewLayoutEvent } from "react-native/Libraries/Components/View/ViewPropTypes";
+import FilterHeaderButton from "./FilterHeaderButton";
+import text from "../../constants/text";
+
+type Props = {
+  visible: boolean,
+  onPress: () => void,
+  animationTime: number,
+  animationDelay: number
+};
+
+type State = {
+  isAnimating: boolean
+};
+
+const DEFAULT_HEIGHT = 120;
+const DEFAULT_FADE_VALUE = 1;
+const DEFAULT_TOP_OFFSET_VALUE = 0;
+
+class ResetAllFiltersButton extends React.Component<Props, State> {
```

Comment:
Do you want to make this a `PureComponent`, so it doesn't rerender all the time?

Reply:
Certainly!